Alloy/commands/compile/parsers/default.js

```diff
@@ -53,6 +53,17 @@ function parse(node, state, args) {
 		args.symbol = CU.generateUniqueId();
 	}
 
+	// find platform specific attributes
+	var platformAttributes = _.filter(_.keys(args.createArgs), key => _.includes(key, ':'));
```

Comment:
Rather than looping through all the keys again in this file, I think it would be better to do this in the initial pass in [compilerUtils.getParserArgs](https://github.com/appcelerator/alloy/blob/784c0a7ea45510953533b1748dae0407316434b8/Alloy/commands/compile/compilerUtils.js#L116). Filtering out any args that aren't for the platform and removing the platform prefix from the attribute.